src/app/window/imp.rs

```diff
@@ -107,14 +107,14 @@ impl WidgetImpl for Window {
 }
 impl WindowImpl for Window {
     // Save window state right before the window will be closed
-    fn close_request(&self) -> Inhibit {
+    fn close_request(&self) -> Propagation {
         // Save window size
         self.obj()
             .save_window_state()
             .expect("Failed to save window state");
 
-        // Don't inhibit the default handler
-        Inhibit(false)
+        // Don't invoke the default handler
+        Propagation::Stop
```

Comment:
https://gtk-rs.org/blog/2023/08/28/new-release.html#replacement-of-glibcontinue--glibinhibit

```
glib::Inhibit(false) -> glib::Propagation::Proceed
```